<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hueApp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/e27_on_waca.svg</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="adressLineEdit">
         <property name="text">
          <string>http://localhost</string>
         </property>
         <property name="placeholderText">
          <string>http://localhost:8080</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="apiLineEdit">
         <property name="text">
          <string>apikey</string>
         </property>
         <property name="placeholderText">
          <string>API Key</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="LightsForm" name="lightsTab">
          <attribute name="title">
           <string>Lights</string>
          </attribute>
         </widget>
         <widget class="MyGroupsForm" name="tab">
          <attribute name="title">
           <string>Custom Groups</string>
          </attribute>
         </widget>
         <widget class="LightsForm" name="groupTab">
          <attribute name="title">
           <string>Groups (Hue Bridge)</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="connectButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionAutofetch"/>
   <addaction name="actionAmbilight"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refreshes all</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAutofetch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autofetch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAmbilight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ambilight</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LightsForm</class>
   <extends>QWidget</extends>
   <header>lightsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyGroupsForm</class>
   <extends>QWidget</extends>
   <header>mygroupsform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
